sql/core/src/test/resources/sql-tests/analyzer-results/pipe-operators.sql.out

```diff
@@ -716,7 +716,22 @@ org.apache.spark.sql.AnalysisException
 
 
 -- !query
-select * from t where first_value(x) over (partition by y) = 1
+table t
+|> where sum(x) over w = 1
+   window w as (partition by y)
+-- !query analysis
+org.apache.spark.SparkException
+{
+  "errorClass" : "INTERNAL_ERROR",
+  "sqlState" : "XX000",
+  "messageParameters" : {
+    "message" : "Cannot find main error class 'NOT_ALLOWED_IN_PIPE_OPERATOR_WHERE.WINDOW_CLAUSE'"
```

Comment:
this means something is wrong... We need to add the error class in `error-conditions.json`